samples/unit-tests/scroller/multinav-dynamic/demo.js

```diff
@@ -188,10 +197,52 @@ QUnit.test('Update options', function (assert) {
         'Navigator series is first series'
     );
     chart.update({
+        xAxis: {
+            labels: {
+                enabled: false
+            }
+        },
         navigator: {
-            baseSeries: 1
+            margin: 50,
+            baseSeries: 1,
+            handles: {
+                backgroundColor: '#ff0000',
+                height: 30,
+                width: 30,
+                symbols: ['circle', 'circle']
+            }
         }
     });
+    assert.strictEqual(
+        redrawsAmount,
+        1,
+        'Updating multiple navigator options should trigger only one redraw.'
+    );
+
+    const handle = chart.navigator.handles[0];
+    assert.strictEqual(
+        handle.attr('d'),
+        'M -0.9999999999999991 30 A 15 15 0 1 1 -0.9998 29.999999998666667 Z',
```

Comment:
I'm curious about the best way to test it. Perhaps we can use SVGRenderer to generate the path and then compare it? It's just when someone changes/improves rendering shapes, then they will have to fix this test. Our target is not to test if the path is exactly the same, but if the shape is correct 🤔

Reply:
I though about it, but then I figured "comparing the path instead of the generated symbol would be better performance wise", so I went with that approach. 

I can change it to symbol (just need to grab the x/y/width/height to match the navigator handle), unless you have a better idea to compare the symbol 🤔